### DISCUSSION TITLE
Efficiency per speed level

### DISCUSSION THREAD

**Discussion by GtHbbrr** [teslamate-org/teslamate#922] [Ideas]
**What would you like to be added**:

On the efficiency dashboard I would like to see the average Wh/km per speed level. 

**Why is this needed**:
efficiency at for example high way speeds on long distance is a subject that is discussed a lot but it is hard to find this efficiency in the car’s statistics

**Comment by riesi77** [comment]
Sounds like an interesting idea - however I am not sure if there is this doable on the plain current speed level, because speed level changes very fast. On top of that going up or down a hill also may make a difference. As well as the temperature.
What I could imagine is to derive this of the trip's data. Take the average speed during a trip and the consumption (and maybe the average temperature).
